extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ScenarioActivity.scala

```diff
@@ -138,7 +153,9 @@ object ScenarioActivity {
       date: Instant,
       scenarioVersionId: Option[ScenarioVersionId],
       comment: ScenarioComment,
-  ) extends ScenarioActivity
+      result: DeploymentRelatedActivityResult,
+      dateFinished: Option[Instant],
```

Comment:
can the date be placed in the Success case class of the DeploymentResult?

Reply:
Done, but inside the DeploymentResult trait - it is present for both Success and Failure, because it is set when operation is marked as finished.